script.js

```diff
@@ -0,0 +1,184 @@
+
+const display = document.querySelector('#display');
+const buttonList = document.querySelectorAll('button');
+const operBtns = document.querySelectorAll('.oper');
+const backspace = document.querySelector('#backspace');
+let userEntry = [];
+
+
+window.addEventListener('keydown', event => {
+    const key = event.key;
+
+    buttonList.forEach(button => {
+        if (key === button.value) {
+            userEntry.push(key);
+            (key === '/')
+                ? display.textContent += '÷'
+                : display.textContent += key;
+        };
+    });
+
+    if (key === 'Enter') {
+        const result = convertExpression(userEntry);
+        userEntry = result;
+        display.textContent = result.join('');
+    } else if (key === 'Escape') {
+        userEntry = [];
+        display.textContent = '';
+    } else if (key === 'Backspace') {
+        userEntry.pop();
+        display.textContent = display.textContent.slice(0, -1);
+    };
+});
+
+buttonList.forEach(button => {
+
+    button.addEventListener('click', event => {
+        const btn = event.target;
+        buttonColor(btn);
+
+        if (btn.classList.contains('alterDisplay')) {
+            if (btn.id === 'cancel') {
+                display.textContent = '';
+                userEntry = [];
+            } else if (btn.id === 'backspace') {
+                display.textContent = display.textContent.slice(0, -1);
+                userEntry.pop();
+            } else {
+                const result = convertExpression(userEntry);
+                userEntry = result;
+                display.textContent = result.join('');
+            };
+        } else {
+            display.textContent += btn.textContent;
+            userEntry.push(btn.value);
+        };
+    });
+});
+
+function buttonColor(button) {
+    operBtns.forEach(btn => btn.style.color = 'white');
+
+    if (button.classList.contains('oper')) {
+        button.style.color = 'orange'
+    };
+};
```

Comment:
style pojint, dont have to follow it 

if(!button.classList.contains...)return 

invert your conditions and return early where possible, means if you do need additional conditions after that point, they wont be nested and easier to read